woocommerce/woocommerce-ios
# [Mobile Payments] Add collect payment button to payments view

Closes: #7361 


### Description

With this PR we add a new row to the IPP Menu screen so the users can collect a payment easily. Once it is tapped, the simple payments amount flow is triggered.
To avoid duplicated code, I create a new class `SimplePaymentsAmountFlowOpener` that encapsulates the logic to start the flow. This class is used in Orders (`AddOrderCoordinator`) and `InPersonPaymentsMenuViewController` (our case here). As in other tasks of this project, the new button is hidden behind the feature flag.
As you can see in the video below, the Collect Payment row is added into its own section with Title header "Actions". We have a thread on Slack to agree on the best approach here: p1659538111766249-slack-C0354HSNUJH

### Testing instructions

- Open the app
- Go to the Menu tab screen
- Tap on Payments
- On the shown IPP Menu, tap on Collect Payment
- Make sure that simple payment can be collected in the three types: Cash, Card, Share Payment Link

Note: Since the IPP Onboarding is not implemented yet, it could be that the check fails and an error screen is shown after tapping on Payments in Menu. In that case, you can test the task here by going to Menu, settings icon, In-Person Payments.

### Screenshots


https://user-images.githubusercontent.com/1864060/182644520-38eee04a-bc49-4fb5-bfe5-e92d2f9399aa.mp4

---
